src/markdown.c

```diff
@@ -1413,6 +1413,25 @@ prefix_quote(uint8_t *data, size_t size)
 	return 0;
 }
 
+static size_t
+prefix_blockspoiler(uint8_t *data, size_t size)
```

Comment:
I think it couldn't hurt, it won't match the surrounding code but hell, gotta start somewhere. :)